tasks.py

```diff
@@ -22,6 +24,11 @@ def genspider(ctx, name, domain):
     test_filename = _gen_tests(name, domain)
     print('Created {0}'.format(test_filename))
 
+    if start_urls:
+        html_filenames = _gen_html(name, start_urls)
```

Comment:
Unfortunately `|` gets read as the pipe command in bash... I could look into seeing if there's a way around that. Oh! maybe we could do commas, and then search for the string `,http://`